.github/workflows/changelog.yml

```diff
@@ -27,15 +27,15 @@ jobs:
       if: startsWith(steps.changelog.outputs.changelog, '####')
       run: |
         # Get release id
-        RELEASE_ID=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+        RELEASE_ID=$(curl -s -H "Authorization: token ${{ secrets.RASABOT_GITHUB_TOKEN }}" \
```

Comment:
Why did we change it here?

Reply:
Thanks for pointing this out! Right we could use `secrets.GITHUB_TOKEN` here as it has enough permission in this case. And it is also safer to use`secrets.GITHUB_TOKEN`. Will revert it back to `secrets.GITHUB_TOKEN`.